src/lib/frontend/components/ProductCard.svelte

```diff
@@ -57,46 +55,50 @@
                 salePrice={salePrice}
         />
     </div>
-    <div
-            class={`absolute bottom-0 right-0 w-full flex justify-end`}>
-        <div
-                class={`filter backdrop-blur-sm bg-green-300/40 font-sans ${expanded ? 'w-full min-h-[4.2rem] justify-between' : 'w-28 h-[2.5rem] justify-center'} flex items-center text-lg`}
-                role="region"
-        >
-            {#if expanded}
-                <div class={`w-8/12 font-light text-xl px-2 text-wrap overflow-hidden text-overflow: ellipsis;`}
-                     in:scale={{ duration: 500, opacity: 0.5, start: 0.5, easing: quintOut }}>
-                    {name}
-                </div>
-            {/if}
-            <div class={`${expanded ? 'w-4/12' : 'w-full'} flex flex-col`}>
-                {#if (percentage && expanded)}
-                    <div class="w-full flex-col items-center justify-center text-xl font-semibold">
-                        ${getPriceFirstPart(salePrice)}
-                        <sup>{getPriceSecondPart(salePrice)}</sup>
-                        {#if (isSoldByQuantities)}
-                            <span class="text-m block text-right pr-1"><sub>/ {getAbbreviatedUnit(unit)}</sub></span>
-                        {/if}
-                    </div>
-                {/if}
-                <div
-                        class={`w-full items-center justify-center ${expanded ? "flex-col" : "flex"} ${expanded && percentage ? "line-through text-lg font-light" : "text-xl font-semibold"}`}>
-                    <span class="block">${getPriceFirstPart(regularPrice)}<sup>{getPriceSecondPart(regularPrice)}</sup></span>
+
+    <div class="w-full flex-col justify-start items-start gap-2.5 flex">
+        <div class="w-full justify-between items-center flex">
+            <div class="text-stone-900 text-xs font-normal pl-1.5 overflow-hidden max-h-[35px]">{name}</div>
+
+            {#if (!percentage)}
+                <div class="text-center text-teal-950 text-15 font-bold flex ml-3">
+                    CAD${getPriceFirstPart(regularPrice)}.{getPriceSecondPart(regularPrice)}
                     {#if (isSoldByQuantities)}
-                        <span class="{expanded ? 'text-m block text-right pr-1' : 'text-sm'}"><sub>/ {getAbbreviatedUnit(unit)}</sub></span>
+                        <span>/{getAbbreviatedUnit(unit)}</span>
```

Comment:
Abbreviated unit can be long. for instance Portion. If we remove <sub>, will there still be enough room?

Reply:
![image](https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/assets/7751678/0e98c91e-38e5-4a5b-9170-a7669f4ffa16)

Reply:
Very nice, thanks @AdderlyPM